src/CommandingOfficers/Modifiers/CODamageModifier.java

```diff
@@ -15,7 +15,7 @@ public CODamageModifier(int percentChange)
   @Override
   public void apply(Commander commander)
   {
-    for( UnitModel um : commander.unitModels )
+    for( UnitModel um : commander.landModels )
```

Comment:
Looks like `CODamageModifier`s cannot apply to air and sea units.

Reply:
I think my preference would be to have a unified model array that various applications can filter, but I dunno how we could easily sort them out again.
This was a stopgap so I didn't have to think more about it since I wanted to move on.

Reply:
Hm. We could consider adding a 'domain' or 'category' attribute to `UnitModel`, and filter on that. We may want to introduce unit attribute flags at some point anyway. Up to you if that's in the scope of this pull request; we can just make an issue and get it later.